labs/scorex.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+title: ScoreX
+parent: Labs
+---
+# Lab Name
+ScoreX
+
+# Short Description
+
+ScoreX is a modular blockchain framework built with flexibility in mind to accelerate the design of experimental DTL technologies. ScoreX is the hyperflexible answer to the Fabric and Sawtooth frameworks. ScoreX is developed and maintained by IOHK and the founders of Ergo under the CC0 1.0 license.
+
+# Scope of Lab
+
+Hyperledger products are categorized in four buckets:
+- Distributed ledgers
+- Domain-specific
+- Libraries
+- Tools
+
+ScoreX would fit into the Distributed ledgers as a new framework designed for modularity and experimentation with blockchain technologies. In specific:
+- Sawtooth separates the core system from the application domain.
+- Iroha presents a modular design targeted to IoT projects.
+- Indy focuses on digital identities.
+- Fabric targets general development of applications or solutions and uses a quite rigid design.
+- Burrow focuses on simplicity but still provides the end user a range of interesting choices.
+- Besu supports Ethereum based chains and is trying to make it easy for enterprises to adopt it.
+
+This six-member family is very comprehensive for enterprise needs, but none of them is targeting a very specific (and important) department: Research and Development. ScoreX solves that gap. ScoreX is modular as the frameworks above, but it is built with a high degree of flexibility allowing R&D departments to prototype their own consensus algorithm, transaction design, block design, ledger state management, memory pool structure and ledger rules.
+
+ScoreX is an effective tool for rapid development of new blockchain protocols. While it is still maturing, projects can be launched quickly and effectively. However, this is a tool for experienced developers as it requires real development to reach its full impact. ScoreX can be an effective product for prototyping, proofs-of-concept, or first releases of implementations. The design and architecture is flexible and ScoreX has good potential for future development
+
+Since its inception, ScoreX has tried to explain the framework in a language that appeals to the researchers, as seen in the [ScoreX whitepaper](https://github.com/ScorexFoundation/ScorexTutorial/blob/master/scorex.pdf). The library is built in a functional language, which allows a closer relation between implementation and research. Finally, it is important to note that Blockchain adoption will happen through R&D departments in many enterprises. It is for all these reasons that IOHK believes ScoreX will bring differentiated and important value to Hyperledger.
+
+
+# Initial Committers
+
+- https://github.com/kushti
+- https://github.com/catena2w
+- https://github.com/ceilican
+- https://github.com/oskin1
+- https://github.com/Tolsi
+- https://github.com/daron666
+
+# Sponsor
+
+- Hart Montgomery (hmontgomery@fujitsu.com)
```

Comment:
@hartm  : Can you confirm sponsorship?

Reply:
Yes!  Sorry for the late response.

Reply:
So as far I can see we have:
1. Small issue (DTL->DLT)
2. Apache 2.0 license for existing code
3. DCO signoff for existing code
Please confirm  @ahmadalkabra, @romainPellerin